### DISCUSSION TITLE
Disappointing - Unable to find guide or documentation on deploying to VPS/Shared Hosting Servers

### DISCUSSION THREAD

**Discussion by murugappanrm** [nuxt/nuxt#16386] [General]
Please let us apologize upfront for venting our frustration. Its has to be done otherwise the developers of the framework may think every thing is perfect about their products and some even get emotional.

It is really, really disappointing to see that all these frameworks, Nuxt is no exception, do not document the methods to deploy the app to a VPS Hosting Server or Shared Hosting Server such as Hostinger (I am not fan of them). We are unable to see a step by step guide on deploying to such servers within the framework documentation or in the internet. The reason we feel is that the documentation is written with only the super-hero developers in mind while completely forgetting newbies coming into the world of vue products. 

I know the developers of these frameworks always tell us off by saying "Are you a sponsoring user" and "how much work we have put in to this...". We are newbies who are just individuals who are trying survive in our jobs.  You are trying make money while we newbies are trying to earn some. So please help us to help you.

**Comment by TheAlexLichter** [comment]
Instead of venting, constructive criticism is always appreciated.

You definitely do **not** have to be a "super-hero developer" to deploy your Nuxt app on e.g. Vercel or Netlify. It is as simple as registering, link it up via GitHub and **that's it**.

You can always statically generate HTML and serve it on **any** shared host by just uploading it. Also, **any** platform supporting Node.js applications can run the outcome of `nuxt build` with the default `node-server` Nitro preset, like you would do it with any other Node application.

PS: This is all open-source and you are more than happy to fiddle around to make it work for the platform you want to use it for, and then submitting a PR to the documentation.

**Reply by murugappanrm** [reply]
@manniL 

This is not criticism, bro. This is frustration as all avenues to get help (including Nuxt creators) seem to fall on the floor. Btw, we dont fiddle with the software as we are business solution developers and our primary focus is to deliver business solutions. We keep the software as is to avoid all the maintenance headaches. I am sure you understand that as business solution developers we more things to worry about instead of fiddling with software.

**Comment by HigherOrderLogic** [comment]
I'm not too similar with deploying Nuxt on VPS but maybe I'll give it a shot. Since VPS is just another machine/desktop, maybe what you need is only to install Node on those machines and then upload the code inside `.output/public` folder to the VPS, finally, use `node .output/server/index.mjs` to run the server (assuming you are using `node-server` preset). This can be simplify by using Docker image.
